src/zkAppWorker.ts

```diff
@@ -51,31 +52,31 @@ setInterval(() => {
       });
     })
     .catch((err) => {
-      console.warn("Update queue error:", err);
+      logger.warn("Update queue error:", err);
```

Comment:
Shouldn't this be `.error()` ?

Reply:
@silimarius this was already a ``` console.warn ``` do you want to keep it an warn or make it an error?